LoopCore/LoopSettings.swift

```diff
@@ -8,6 +8,22 @@
 import HealthKit
 import LoopKit
 
+public enum DosingStrategy: Int, CaseIterable {
+    case tempBasalOnly
+    case automaticBolus
```

Comment:
[nit] doesn't `automaticBolus` mean `automaticDosing` (which includes temp basal and bolus)? automaticBolus reads like temp basal is not included.

Reply:
This is the name of the dosing *strategy*. Even though this dosing strategy can modulate temp basals, by setting a rate less than scheduled, the users refer to this strategy as "automatic bolus". `automaticDosing` would apply to both strategies, as this setting specifies the strategy for automatic dosing.